core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -313,7 +313,10 @@ var TopicsAndSkillsDashboardPage = function() {
       editor, 'Explanation Editor takes too long to appear');
     var skillReviewMaterialInput = editor.element(by.css('.oppia-rte'));
     await action.click(
-      'Skill review material input', skillReviewMaterialInput);
+      'Skill Review Material Input', skillReviewMaterialInput);
+    var rteToolbar = element(by.id('cke_editor1'));
+    await waitFor.visibilityOf(
```

Comment:
We use RTE in few other places too, right? Do we need to do a similar thing in all the places where RTEs are used?

Reply:
I think like this is a bug and it needs a fix in the source code. If an input component is shown in the UI, the user should be able to enter text into it and expect it to save -- they should not have to worry about the RTE bar appearing before they type in something. Can you please file a bug for this and assign it to the LaCE team?

Reply:
I tried adding a `waitFor()` in the RTE code in `forms.js`, but this caused the tests to fail. For the time being, I've added the optional parameter `isRte` to `action.sendKeys()` -- in the future, we could use this. Right now, I think we should prioritize getting this PR merged since the create skill button flake is one of the most common flakes.